python/nano/src/bigdl/nano/deps/neural_compressor/core/quantization.py

```diff
@@ -49,6 +50,11 @@ def __init__(self,
                                      allows relative accuracy loss: 1%. accuracy_criterion = {
```

Comment:
may update this comment, for my understanding, `'relative': 0.1` means accuracy >= 10% * original_accuracy ? and `'absolute: 0.99'` means accuracy > 0.99 ?

Reply:
For exmaple, if `'absolute: 0.99'` and `higher_is_better` is True, this requires original metric value subtract current metric value be smaller than 0.99. For `'relative'`, it's similar.
Have revised corresponding comment.

Reply:
Oh, I understand. But why we set it to `relative: 0.99` by default, this allows 99% metric value loss ? INC's default value is `relative: 0.01`, why we use 0.99 ?

Reply:
I think `relative: 0.99` actually means no tunning, and we set this value to ensure quantization always works and don't raise error like `can't find model meeting accuracy_criterion`.
I have tried` relative: 0.01` before and some ut of chronos will failed as can't get quantized model meet such strict accuracy limit.

Reply:
well, but I think 0.99 is too relaxed, if the metric value drops to 1%, I think this optimization is pointless. Have we tried other value? such as 0.9, 0.8 ?

Reply:
`if the metric value drops to 1%, I think this optimization is pointless.`
Of course, in most cases, this will not happen.
However, in our uts, we always use random data for quantization, so that the accuracy of quantized model has no guarantee.
I will try 0.8 but such value will import a potential risk for action failing.

Reply:
> I will try 0.8 but such value will import a potential risk for action failing.

We can set its default value to 0.01, as for the UT, we can pass 0.99 to this parameter

Reply:
Or there is another solution, we set default value to 0.01 as INC's, and adjust related failed uts. How do you think of?

Reply:
well, we have the same idea